test/e2e-batch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: inventory-e2e-tests
+spec:
+  template:
+    spec:
+      containers:
+        - name: inventory-e2e-tests
+          image: localhost/inventory-e2e-tests:latest
+          env:
+            - name: INV_GRPC_URL
+              value: "inventory.multicluster-global-hub.svc.cluster.local:9081"
+            - name: KAFKA_BOOTSTRAP_SERVERS
+              value: "kafka-kafka-bootstrap.multicluster-global-hub:9092"
+      restartPolicy: Never
+  backoffLimit: 1
```

Comment:
won't this need to mount the kafka creds?

Reply:
yes, I couldn't get the global hub set up through the script `e2e_setup_mcgh.sh`. I wasn't sure what configurations I need to configure for kafka.